index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <section>
+        <header id="navbar">
+            <a href="#" class="my-logo">My Logo</a>
+            <nav>
+                <img src="images/haburger_menu.svg" alt="menu">
+            </nav>

```

Comment:
As requested by the previous reviewer, kindly fix this image path, as shown below, it is currently broken:
![image](https://user-images.githubusercontent.com/42799579/198428855-738c9fe9-ce2e-41fd-860e-63b5b7e8728c.png).
This is happening because you are referencing an images folder but your project doesn't contain any folder with images. Keep in mind that if an image is on your computer, you should put it inside the project's folder so others can also see it 😄.